.github/workflows/dockerimage-nvcc-11-ubuntu-mpich.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         message("::set-output name=timestamp::${current_date}")
     - uses: actions/cache@v1
       env:
-        cache-name: ubuntu-nvcc-11.2-cache
+        cache-name: ubuntu-11.2.0-cache
```

Comment:
The cache name should probably specify that it's for nvcc

Reply:
@PhilMiller Ok, Fixed.